.buildkite/build-js.sh

```diff
@@ -11,5 +11,8 @@ restore_cache "$CACHEKEY"
 echo '--- :react: Generate JS bundles'
 make bundle-ci
 
+echo '--- :git: Ensure git repo is clean'
+[ -z "$(git status --porcelain)" ]
```

Comment:
@jkmassel The message for the commit that added this (thank you for writing it!) says:

> This ensures we didn’t accidentally forget to deploy assets

Without that, it would have taken me a while to understand why this check was here.

How do you feel about printing an error message?

```suggestion
echo '--- :git: Ensure git repo is clean'
if [[ ! -z "$(git status --porcelain)" ]]; then
    echo "Bundling the library generated changes that were not tracked in the Git repository. Please track those changes in a new commit."
    exit 1
fi
```